### PR TITLE
Accessible dropdown

### PR DESCRIPTION
**Pull request by dartamonov-vertex**
Added keyboard support to the component.
* Focus is passed to the dropdown menu when it gets open
* Menu can be closed with TAB and ESC keys
* Focus is passed back to the trigger on menu select

Note: two existing tests in basic.test.js are failing in master branch

### CODE REVIEW

**Comment by dartamonov-vertex** [comment]
@MadCcc @afc163 @zombieJ @howel52 can you guys review this PR, please. We are trying to get this accessibility fix to the Ant Design.

**Comment by MadCcc** [comment]
Can you add some test for it？

**Comment by dartamonov-vertex** [comment]
@MadCcc accessibility unit test was added

**Comment by MadCcc** [comment]
We have updated CI of this repository, could you please pull the latest code in master?

**Comment by dartamonov-vertex** [comment]
Updated with the latest from master

**Comment by afc163** [comment]
图片

**Comment by dartamonov-vertex** [comment]
CI uses node 12, which is going to be end-of-life'd in 3 months: https://nodejs.org/en/about/releases/
I'm not getting any of the CI errors locally with node 14. 
So I updated CI node version to 14. My expectation is it should fix all CI errors.

**Comment by dartamonov-vertex** [comment]
I think the problem is although the CI node node version is updated in my PR, CI that checks my PR is used from master and therefore it still uses Node 12. 
Node version has to be updated in a separate PR - https://github.com/react-component/dropdown/pull/191 and this PR have to be re-validated again.

**Comment by MadCcc** [comment]
image
The version is v14.x

**Comment by MadCcc** [comment]
Please rebase master

**Comment by dgreene1** [comment]
@MadCcc @yhy-1 rebased to master.

File: src/Dropdown.tsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
-import * as React from 'react';
-import type { TriggerProps } from 'rc-trigger';
-import Trigger from 'rc-trigger';
-import classNames from 'classnames';
+import * as React from "react";
```
Comment (afc163) [inline src/Dropdown.tsx:1]: We should use `'` instead of `"`.

**Comment by MadCcc** [comment]
Error in test case.
image

**Pull request merged**